dev-packages/browser-integration-tests/suites/public-api/init/built-pkg/subject.js

```diff
@@ -75,6 +58,6 @@ Sentry.withScope(scope => {
 
 var xhr = new XMLHttpRequest();
 xhr.onload = () => console.log('loaded'); // This throws error
-// xhr.addEventListener("load", () => console.log('loaded')); This does not throw error
+xhr.addEventListener('load', () => console.log('loaded')); // This does not throw error
```

Comment:
l/feel free to ignore: do we need this log?

Reply:
We can remove it!